include/mantella_bits/orbitalMechanics.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+// C++ standard library
+#include <utility>
+#include <vector>
+
+// Armadillo
+#include <armadillo>
+
+namespace mant {
+  namespace itd {
+
+    // from pykep:
+    // ref_mjd2000(epoch(2451545.0,epoch::JD).mjd2000()
+    //inline double jd2mjd2000(const double & in){
+    //return ( in - 2451544.5 );
+    //}
+    // => 2451545.0 - 2451544.5
+    const double refModifiedJulianDay2000 = 0.5;
+
+    double lambda;
+
+    const double au = 149597870691.0;
+    const double muSun = 1.32712440018e20;
+
+    const int lambertMaxIterations = 13;
+    const double lambertTolerance = 1e-13;
+
+    double degreeToRadian(
```

Comment:
Did we actually need this? While I can think of scenarios where this can become useful, it is only used to convert keplerValuesPreCalculationVector(2) to keplerValuesPreCalculationVector(5) one time - which are only linearly adjusted beforehand.

And since the calculation inside `degreeToRadian` is also linear, we could also just specify the third to sixth Kepler value based on radians.